From e06ca6846962a5048b80ab9d4eb140465e5efbc7 Mon Sep 17 00:00:00 2001
From: John Robson <John.Robson@USP.br>
Date: Fri, 2 Mar 2012 19:20:47 -0300
Subject: [PATCH] Apenas um teste ;)

---
 run.sh |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/run.sh b/run.sh
index 7871285..76708da 100755
--- a/run.sh
+++ b/run.sh
@@ -40,4 +40,5 @@ killall -9 ${deploy}.wt
 echo "************************* starting ${project} *************************";
 ${www}${deploy}.wt --approot ${www} --docroot ${www} --http-addr ${httpaddr} --http-port ${httpport}
 
+# alterando
 #--deploy-path ${www}${project}
-- 
1.7.9

